File: src/taeeun/week4/멀쩡한 사각형.js
function solution(w, h) {
    var answer = 1;
    
    const gcd=(a, b)=>{
        const mod=a%b;
        if (mod===0) return b;
        return gcd(b, mod);
    }

    return w*h-(w+h-gcd(w,h));
}

File: src/jimin/week4/멀쩡한 사각형.py
# https://luv-n-interest.tistory.com/736
def get_gcd(x, y):
    if (y == 0):
        return x
    else:
        return get_gcd(y, x % y)


def solution(w, h):
    gcd = get_gcd(max(w, h), min(w, h))
    return w * h - (w + h - gcd)


w, h = 8, 12
print(solution(w, h))
